### DISCUSSION TITLE
Is there a php v 7.2 version? - don't really want to have to upgrade my php

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [spatie/laravel-settings#43] [General]
Is there a php v 7.2 version? - don't really want to have to upgrade my php

**Comment by ragingdave** [comment]
I don't want to sound judge-y, but 7.2 is an EOL version of PHP, so it wouldn't really make sense to make a version compatible with an EOL version, but that's just my two-cents.

**Reply by michaeljcoyne** [reply]
No worries - will evaluate locally and if it is something I want to use will upgrade my servers. Thanks

**Reply by ragingdave** [reply]
I'm not a maintainer or anything so I don't have really any say if it happens or not, I was just adding my opinion here.

**Reply by rubenvanassche** [reply]
Hi @michaeljcoyne,

We at Spatie believe keeping up with the newest PHP versions is the way to go, that's why you won't find a 7.3 or 7.2 version of this package. We'll probably upgrade the whole package to php 8.0 with the next major version 😄

**Reply by michaeljcoyne** [reply]
Fair enough - just means there are repuccusions with other plugins which I experienced when upgrading.
